fendermint/app/src/cmd/genesis.rs

```diff
@@ -246,3 +252,64 @@ fn set_ipc_gateway(genesis_file: &PathBuf, args: &GenesisIpcGatewayArgs) -> anyh
         Ok(genesis)
     })
 }
+
+async fn new_child_genesis(
+    genesis_file: &PathBuf,
+    args: &GenesisIpcChildGenesisArgs,
+) -> anyhow::Result<()> {
+    // provider with the parent.
+    let parent_provider = IpcProvider::new_with_subnet(
+        None,
+        ipc_provider::config::Subnet {
+            id: args.subnet_id.clone(),
```

Comment:
What does `id` do? Why isn't the the subnet ID of the _parent_ subnet, within which we can inquire about the child subnets?

Reply:
Bug from my side, already fixed.